kiva/quartz/setup.py

```diff
@@ -60,6 +63,10 @@ def generate_c_from_cython(extension, build_dir):
                                   ],
                         )
 
+    extra_link_args = []
+    for framework in frameworks:
+        extra_link_args.extend(['-framework', framework])
```

Comment:
Why the extra step here? `extra_link_args` will have the same value it got above.

Reply:
Pretty sure there was a reason at some point :-| -- but looks like you're absolutely correct, I'll test out removing this bit and see what happens.